assets/docs/sayannath/american-sign-language/1.md

```diff
@@ -0,0 +1,31 @@
+# Module sayannath/american-sign-language/1
+
+The Image Scene Classification model can classify the images based on the american sign language alphabet. The model can handle upto 29 classes from A to Z and Space, Delete and Nothing as well. 
+
+<!-- asset-path: https://github.com/sayannath/American-Sign-Language-Detection/releases/download/1.0.0/asl_classifier.zip -->
```

Comment:
Could you also upload the file to AWS? github.com/*/download disallows crawling it via its robots.txt policy.

Reply:
@WGierke it is giving error in this.

```
{'sayannath/american-sign-language/1.md': 'File cannot be used by SavedModel: asl_classifier/keras_metadata.pb'}
```

Reply:
Sorry for not catching that earlier - this test should just ensure that no additional files unrelated to the SavedModel are uploaded. Could you remove this file from the archive and try again? Sorry for the inconvenience!

Reply:
Any idea on the new error? Because `saved_model.pb` is an important file.

Reply:
Oh I think the issue is that the files aren't at the top level of the archive. Could you do
`cd asl_classifier && tar -czvf ../model.tar.gz *` and upload that `model.tar.gz` file? I was already wondering why `keras_metadata.pb` was prohibited since it should be fine telling from https://github.com/tensorflow/tfhub.dev/blob/master/tools/validator.py#L168. So if you want, you can add `keras_metadata.pb` again

Reply:
Extracting the archive yields
```
total 2.8M
6036330 4.0K drwxr-xr-x 4  501 staff 4.0K Mar 21 15:23 .
1835033 4.0K drwxr-xr-x 1 root root  4.0K Mar 21 15:47 ..
6036382 4.0K drwxr-xr-x 2  501 staff 4.0K Mar 19 12:55 assets
6036381 4.0K -rwxr-xr-x 1  501 staff  212 Mar 19 12:55 ._assets
6036349 4.0K -rw-r--r-- 1  501 staff  212 Mar 19 12:55 ._keras_metadata.pb
6036350 344K -rw-r--r-- 1  501 staff 344K Mar 19 12:55 keras_metadata.pb
6036353 4.0K -rw-r--r-- 1  501 staff  212 Mar 19 12:55 ._saved_model.pb
6036380 2.5M -rw-r--r-- 1  501 staff 2.5M Mar 19 12:55 saved_model.pb
6036352 4.0K drwxr-xr-x 2  501 staff 4.0K Mar 21 15:48 variables
6036351 4.0K -rwxr-xr-x 1  501 staff  212 Mar 19 12:55 ._variables
```
Could you remove the files that start with `._`? I think this is an OSX-specific thing

Reply:
Cool

Reply:
@WGierke the problem still persists. Can you manually upload the serialised model?

Reply:
Could you give it one more attempt? The following worked for me:
```bash
wget https://presence-bucket.s3.amazonaws.com/server/model.tar.gz
tar -xvf model.tar.gz
cd model && tar -czvf ../model2.tar.gz * && cd ..
tar -xvf model2.tar.gz # Should show that all files are in the top-level directory
```
If this doesn't work, I'll upload it manually.

Reply:
Cool! Will do so.

Reply:
@WGierke Still the problem persists :(

Reply:
Could you replace the asset-path URL with https://storage.googleapis.com/tfhub-modules/tmp/model.tar.gz?

Reply:
Sure!